docker-compose.yml

```diff
@@ -21,6 +21,8 @@ services:
   skyvern:
     image: public.ecr.aws/skyvern/skyvern:latest
     restart: on-failure
+    env_file:
+      - .env
```

Comment:
the `environment` section will overrides whatever is inside the .env. So this might not work as expected as the LLM_KEY is sent under `environment`

Reply:
sg. if the user doesn't have .env file, let's make sure `docker compose` won't break

Reply:
![Screenshot 2025-05-29 at 12 36 04 PM](https://github.com/user-attachments/assets/0c67f654-240a-423a-a99e-cdb55c2a6657)

it does break... so this is risky/confusing.
I'd suggest having a flow where we run `docker compose up` using a script which checks for the existence of the .env file and if it doesn't exist, prompts the user to create it, similar to how the current `./setup.sh` does.